docs/rdataretriever.rst

```diff
@@ -2,7 +2,7 @@
 Using the Data Retriever from R
 ===============================
 
-rdataretriever
+rdataretriever_
```

Comment:
When you run `make html` in the docs folder you get an error.
You need to add and extra = since you add _

Reply:
![image](https://user-images.githubusercontent.com/31954786/77366680-87362180-6d7e-11ea-96c5-e98b9c546728.png)

Reply:
Made the change